css/style.css

```diff
@@ -24,8 +24,52 @@ button a:hover{
   text-decoration: none;  
 }
 
+/*Stying for Tables made with div tags instead of table tags
+Adapted from: 
+http://www.dummies.com/web-design-development/css3/using-the-div-tag-to-create-tables/
+
+and from CSS and HTML generated at: 
+http://divtable.com/generator/
+*/
+.Table {
```

Comment:
its not normally the convention to camelCase css class. normally you do something-like-this. 
If you are going to camel case, the first letter should be lower case.